content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -199,9 +219,11 @@ the webhooks. There are three steps to complete the configuration.
         kind: WebhookAdmission
         kubeConfigFile: "<path-to-kubeconfig-file>"
     ```
+    {{% /tab %}}
+    {{< /tabs >}}
 
 The schema of `admissionConfiguration` is defined
-[here](https://github.com/kubernetes/kubernetes/blob/v1.13.0/staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go#L27).
+[here](https://github.com/kubernetes/kubernetes/blob/v1.17.0/staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1/types.go#L27).
```

Comment:
@liggitt 
- nit: clean up use of `here`, suggested edit:
For more information about `AdmissionConfiguration`, see the [AdmissionConfiguration schema](https://github.com/kubernetes/kubernetes/blob/v1.17.0/staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1/types.go#L27).

- Will this link exist after the 1.17 release?  Currently, not available.

Reply:
done. yes, the link will exist when 1.17 is cut